src/core/lv_obj_event.c

```diff
@@ -18,11 +18,6 @@
 /**********************
  *      TYPEDEFS
  **********************/
-typedef struct _lv_event_dsc_t {
-    lv_event_cb_t cb;
-    void * user_data;
-    lv_event_code_t filter : 8;
-} lv_event_dsc_t;
```

Comment:
@kisvegabor why need define `lv_event_dsc_t` twice here

Reply:
By accident :sunglasses: 
Thanks for removing it.